<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>493</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>493</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DWYT - Get Started</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>6</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/images/icons/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logoTitleLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>183</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Marker Felt&quot;;</string>
    </property>
    <property name="text">
     <string>Don't Waste Your Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>161</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/images/icons/about.png</normaloff>:/images/icons/about.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>259</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Activity</string>
     </attribute>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Progress</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="progressBarTitle">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>23</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 18pt &quot;Helvetica&quot;;</string>
      </property>
      <property name="text">
       <string>PRODUCTIVITY</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="sessionInfoGroup">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="sessionTimeLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;Helvetica&quot;;</string>
       </property>
       <property name="text">
        <string>Reports every</string>
       </property>
      </widget>
      <widget class="QLabel" name="intervalValueLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>15 min</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Manage</string>
     </attribute>
     <widget class="QPushButton" name="statisticButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>1</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Press this button to show your curent activity</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 13pt &quot;Helvetica&quot;;</string>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/images/icons/statistic.png</normaloff>:/images/icons/statistic.png</iconset>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>All sessions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Press to set up your custom preferences</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/images/icons/settings.png</normaloff>:/images/icons/settings.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>3</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="intervalEdit">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>34</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="setNewInterval">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>30</y>
         <width>61</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Enter new interval</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="birdProgress">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/images/icons/bird1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
